contracts/BufferLib.sol

```diff
@@ -52,6 +52,7 @@ library BufferLib {
     uint64 newCursor = _offset;
     // Deal with relative offsets
     if (_relative == true) {
+      require(newCursor + _buffer.cursor > newCursor && newCursor + _buffer.cursor > _buffer.cursor, "The sum newCursor caused an overflow");
```

Comment:
I think checking only against one of them is sufficient. Thus just leave the first check in this require